extensions/gitlab/src/components/project_search.tsx

```diff
@@ -19,6 +20,12 @@ export function ProjectSearchList(): JSX.Element {
       onSearchTextChange={setSearchText}
       isLoading={isLoading}
       throttle={true}
+      searchBarAccessory={
+        <List.Dropdown tooltip="Scope" onChange={setScope}>
```

Comment:
Let's store the value of the dropdown before next rendering of better user experience.
```suggestion
        <List.Dropdown tooltip="Scope" onChange={setScope} storeValue>
```

Reply:
Thanks, i was looking for this functionality through `LocalStorage`, didn't know about the `storeValue`!